watsong/static/js/webapp.js

```diff
@@ -126,7 +126,16 @@ function showPlaylist() {
     GLOBAL.setPlaylistId(playlistId);
   });
 }
-
+// Display name of query above playlist
```

Comment:
Since we don't need this, let's remove it so we don't wonder what it's for later on.